build.sbt

```diff
@@ -88,21 +88,8 @@ resolvers ++= Seq(
   "Maven Central" at "https://repo1.maven.org/maven2/",
 )
 
-def javaCheckstyleSettings(configLocation: File) = Seq(
```

Comment:
no need to remove this. We should rather check the conflicting checkstyle rules rather than removing everything. Having checkstyle still has value, so we need to double-check the conflicting checkstyle rules